learning-another-language/fizzbuzz.md

```diff
@@ -0,0 +1,258 @@
+| --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| `const fizzBuzz`                                    | This _declares_ a variable named `fizzBuzz`. This specific type of variable is a `const` variable. The variable's name is in camelCase. |
+| `=`                                                 | This is the assignment operator. This will assign whatever value is on the right-side to the variable on the left side, `fizzBuzz`.     |
+| `function`                                          | The keyword to begin the definition of a function                                                                                       |
+| `(num)`                                             | This section defines the function parameters. This function has one parameter, named `num`.                                             |
+| `{`                                                 | This opening curly brace (`{`) begins the function body.                                                                                |
+
+## Conditionals, Comparisons, and Returns
+
+Now, let's dive into this fragment of code inside the function body:
+
+<!-- prettier-ignore-start -->
+```js
+if (num % 15 == 0) {
+  return 'FizzBuzz';
+} else if (num % 3 == 0) {
+  return 'Fizz';
+}
+```
+<!-- prettier-ignore-end -->
+
+| <div style="min-width:200px;"> Piece of Code </div> | Notes                                                                                                                                                                 |
+| --------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `if`                                                | The keyword to begin an if-statement.                                                                                                                                 |
+| `(...)`                                             | We put the conditional expression in parentheses `()`. If the expression evaluates to `true`, then the if-statement will execute the block of code.                   |
+| `num % 15 == 0`                                     | The conditional expression. It reads the parameter `num` and uses the mod operator `%`. It checks if `num % 15` is equal to `0` using equality comparison, with `==`. |
+| `{`                                                 | We begin the block of code to execute with an open curly brace `{`. **Everything inside the code block is indented one level**.                                       |
```

Comment:
yes, we should indent. but callout that this is only for readability (explicitly chose not to mention python by name).
```suggestion
| `{`                                                 | We begin the block of code to execute with an open curly brace `{`. **Everything inside the code block is indented one level**. In JavaScript, the indentation has no semantic meaning, but it still helps us understand the code intent.                                      |
```